tests/tools/time.sh

```diff
@@ -22,6 +22,10 @@ wait_for() {
 	return 1
 }
 
+wait_chunkservers() {
+	sleep 5
```

Comment:
In addition to adding a small comment about what this function is used for, it would be good to specify why 5 was selected as the chunkservers' wait value.